pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -2438,36 +2442,6 @@ class ShapePropagator : public PropertyPropBase {
         node->output()->setType(type->withSymbolicShapes(newSizes));
       }
       return true;
-    } else if (
```

Comment:
Why delete this part directly? what's the difference between this part and the above part?

Reply:
This function will use input complete shape info to infer the node's output shape. For `aten::slice`, they make no difference here.